repos/system_upgrade/el7toel8/actors/pcidevicesscanner/tests/test_pcidevicesscanner.py

```diff
@@ -0,0 +1,122 @@
+from leapp.snactor.fixture import current_actor_context, current_actor_libraries
+from leapp.models import PCIDevices, PCIDevice
+
+
+def test_parse_pci_devices(current_actor_libraries):
+    from leapp.libraries.actor.pcidevicesscanner import parse_pci_devices
```

Comment:
You can import the library in the top of the file. It doesn't need to be imported within a test function anymore.

Reply:
At least on my environment moving the import to top of the file is failing. Any idea what cna be wrong?

```
(leapp-devel) [vagrant@localhost pcidevicesscanner]$ pwd
/home/vagrant/work/leapp-repository/repos/system_upgrade/el7toel8/actors/pcidevicesscanner
(leapp-devel) [vagrant@localhost pcidevicesscanner]$ ls -la libraries/
total 8
drwxrwxr-x. 2 vagrant vagrant   63 Jan 17 22:34 .
drwxrwxr-x. 4 vagrant vagrant   85 Jan 17 22:22 ..
-rw-rw-r--. 1 vagrant vagrant 1801 Jan 17 22:32 pcidevicesscanner.py
-rw-rw-r--. 1 vagrant vagrant 3117 Jan 17 22:34 pcidevicesscanner.pyc
(leapp-devel) [vagrant@localhost pcidevicesscanner]$ git diff tests/test_pcidevicesscanner.py
diff --git a/repos/system_upgrade/el7toel8/actors/pcidevicesscanner/tests/test_pcidevicesscanner.py b/repos/system_upgrade/el7toel8/actors/pcidevicesscanner/tests/test_pc
index c9420e1..217f67e 100644
--- a/repos/system_upgrade/el7toel8/actors/pcidevicesscanner/tests/test_pcidevicesscanner.py
+++ b/repos/system_upgrade/el7toel8/actors/pcidevicesscanner/tests/test_pcidevicesscanner.py
@@ -1,5 +1,6 @@
 from leapp.snactor.fixture import current_actor_context, current_actor_libraries
 from leapp.models import PCIDevices, PCIDevice
+from leapp.libraries.actor import pcidevicesscanner
 
 
 def test_parse_pci_devices(current_actor_libraries):
(leapp-devel) [vagrant@localhost pcidevicesscanner]$ cd tests/ && pytest
========================================================================== test session starts ===========================================================================
platform linux2 -- Python 2.7.5, pytest-4.1.1, py-1.7.0, pluggy-0.8.1
rootdir: /home/vagrant/work/leapp-repository/repos/system_upgrade/el7toel8/actors/pcidevicesscanner/tests, inifile:
plugins: leapp-0.4.0
collected 0 items / 1 errors                                                                                                                                             

================================================================================= ERRORS =================================================================================
_______________________________________________________________ ERROR collecting test_pcidevicesscanner.py _______________________________________________________________
ImportError while importing test module '/home/vagrant/work/leapp-repository/repos/system_upgrade/el7toel8/actors/pcidevicesscanner/tests/test_pcidevicesscanner.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/home/vagrant/devel/leapp-devel/lib/python2.7/site-packages/six.py:709: in exec_
    exec("""exec _code_ in _globs_, _locs_""")
test_pcidevicesscanner.py:3: in <module>
    from leapp.libraries.actor import pcidevicesscanner
E   ImportError: cannot import name pcidevicesscanner
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
======================================================================== 1 error in 0.09 seconds =========================================================================
(leapp-devel) [vagrant@localhost tests]$ 
```

Reply:
@bocekm Moved the import to the top of the file and tests worked as expected now. So this should be resolved on latest commit :)